renku/ui/cli/run.py

```diff
@@ -305,7 +507,8 @@
     help="Allowed command exit-code.",
 )
 @click.option("--isolation", is_flag=True, default=False, help="Invoke the given command in isolation.")
-@click.argument("command_line", nargs=-1, required=True, type=click.UNPROCESSED)
+@click.option("--file", is_flag=True, default=False, help="Force running of a workflow file.")
+@click.argument("command_line", nargs=-1, metavar="<COMMAND> or <WORKFLOW FILE>", required=True, type=click.UNPROCESSED)
```

Comment:
Should we put this at the end so it's not mixed/hidden in between click.options?